addon/index.js

```diff
@@ -549,7 +549,15 @@ export function changeset(obj, validateFn = defaultValidatorFn, validationMap =
 
         let errors = get(this, ERRORS);
         if (errors['__ember_meta__'] && errors['__ember_meta__']['values']) {
-          delete errors['__ember_meta__']['values'][key];
+          let path = key.split('.');
+          if (path.length === 1) {
+            delete errors['__ember_meta__']['values'][key];
+          } else {
+            let branch = path.slice(0, -1).join('.');
+            let [leaf] = path.slice(-1);
+            let obj = get(errors, `__ember_meta__.values.${branch}`);
+            if (obj) delete obj['__ember_meta__']['values'][leaf];
```

Comment:
Note to self: this may break for triple nested keys, add another test case.

Reply:
#230 Works for nested keys 👍